src/network/providers/ethereum.js

```diff
@@ -540,8 +540,32 @@ ethereum.getKeyValue = async (
     identity,
     key,
     version = 'latest',
-    versionSearchStrategy = 'exact'
+    versionSearchStrategy = 'exact',
+    followCopyFromIkv = false
 ) => {
+    // Process @@copy_from_ikv instruction if followCopyFromIkv is set to true
+    if (followCopyFromIkv) {
+        // self invoke to get the value first but without redirection
+        const value = ethereum.getKeyValue(identity, key, version, versionSearchStrategy, false);
```

Comment:
You'll probably want to `await` this one.

Reply:
yep, thanks!